taxi/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
```

Comment:
According to the checklist, you should use `get_user_model()` instead of `Driver`. This is a code efficiency issue. Using `get_user_model()` is more flexible and safer, because it automatically returns the currently active user model - the custom user model if one is specified, or `User` otherwise. So, if you later decide to use a custom user model, you won't need to change this code.